server/src/routes/auth0.ts

```diff
@@ -0,0 +1,43 @@
+import { Request, Response } from 'express';
+import { ServerCredentials } from '../common/types';
+import { error } from '../utils/log';
+
+export function createAuth0Route(serverCredentials: ServerCredentials) {
```

Comment:
If I understand this correctly, user.mock.ts calls this route to get auth0 token, bypassing the client side auth0 authorization path correct?

We should not add this here since it's only for testing. We don't want customers deploying this route.
Can you make this auth0 request from user.mock.ts instead? You probably need to move the environment variables needed, but I think we already have the tooling for that right?

Reply:
Yes this calls the token route by bypassing the client side auth0 auth path. 

Is it safe to do this in the client side though through `fetch`? I only saw node.js as an option in their docs: https://auth0.com/docs/get-started/authentication-and-authorization-flow/call-your-api-using-resource-owner-password-flow#example-post-to-token-url

Reply:
It's not safe to put this client side _**iff**_ we are doing this outside of testing environment. However, since we're only putting it in the mock file, we can put this in the client. The node.js option they provided in their doc is just an example. As long as we create the same request from the client, this should work.